integration-cli/docker_cli_pause_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestEventsPause(t *testing.T) {
+	name := "testeventpause"
+	out, _, _ := dockerCmd(t, "images", "-q")
+	image := strings.Split(out, "\n")[0]
+	dockerCmd(t, "run", "-d", "--name", name, image, "sleep", "2")
+	dockerCmd(t, "pause", name)
+	dockerCmd(t, "unpause", name)
```

Comment:
Can you use the getPausedContainers() func to actually get the list of paused containers?  I know you check the event log later on but I'd like to actually check the current state (between the pause and unpause) to make sure things are really paused.